### ISSUE TITLE
netty-tcnative-boringssl-static 2.0.4 missing native deps

### ISSUE THREAD

**Issue by carl-mastrangelo** [netty/netty-tcnative#291]
Not sure why, but the shared objects normally present in the maven jar are missing.
Before:
```
Fri Jun 23 04:22:40$ unzip -l ./.gradle/caches/modules-2/files-2.1/io.netty/netty-tcnative-boringssl-static/2.0.3.Final/28d278ac2ae302ae9bdf19958c75b16fefbfae00/netty-tcnative-boringssl-static-2.0.3.Final.jar
Archive:  ./.gradle/caches/modules-2/files-2.1/io.netty/netty-tcnative-boringssl-static/2.0.3.Final/28d278ac2ae302ae9bdf19958c75b16fefbfae00/netty-tcnative-boringssl-static-2.0.3.Final.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2017-06-07 18:43   META-INF/
     1361  2017-06-07 18:43   META-INF/MANIFEST.MF
        0  2017-06-07 18:43   io/
        0  2017-06-07 18:43   io/netty/
        0  2017-06-07 18:43   io/netty/internal/
        0  2017-06-07 18:43   io/netty/internal/tcnative/
        0  2017-06-07 18:43   META-INF/native/
      357  2017-06-07 18:43   io/netty/internal/tcnative/Buffer.class
      743  2017-06-07 18:43   io/netty/internal/tcnative/CertificateRequestedCallback$KeyMaterial.class
      699  2017-06-07 18:43   io/netty/internal/tcnative/CertificateRequestedCallback.class
     9049  2017-06-07 18:43   io/netty/internal/tcnative/CertificateVerifier.class
     2941  2017-06-07 18:43   io/netty/internal/tcnative/Library.class
     3834  2017-06-07 18:43   io/netty/internal/tcnative/NativeStaticallyReferencedJniMethods.class
     1262  2017-06-07 18:43   io/netty/internal/tcnative/SessionTicketKey.class
      185  2017-06-07 18:43   io/netty/internal/tcnative/SniHostNameMatcher.class
     5660  2017-06-07 18:43   io/netty/internal/tcnative/SSL.class
     2985  2017-06-07 18:43   io/netty/internal/tcnative/SSLContext.class
      455  2017-06-07 18:42   META-INF/io.netty.versions.properties
  1988184  2017-06-07 18:43   META-INF/native/libnetty-tcnative-linux-x86_64.so
  2045808  2017-06-07 18:43   META-INF/native/libnetty-tcnative-osx-x86_64.jnilib
  1820672  2017-06-07 18:43   META-INF/native/netty-tcnative-windows-x86_64.dll
        0  2017-06-07 18:43   META-INF/maven/
        0  2017-06-07 18:43   META-INF/maven/io.netty/
        0  2017-06-07 18:43   META-INF/maven/io.netty/netty-tcnative-boringssl-static/
    22467  2017-06-07 18:16   META-INF/maven/io.netty/netty-tcnative-boringssl-static/pom.xml
      131  2017-06-07 18:43   META-INF/maven/io.netty/netty-tcnative-boringssl-static/pom.properties
      249  2017-06-07 18:43   META-INF/INDEX.LIST
---------                     -------
  5907042                     27 files

```

After:
```
Fri Jun 23 04:21:34$ unzip -l ./.gradle/caches/modules-2/files-2.1/io.netty/netty-tcnative-boringssl-static/2.0.4.Final/b02dee1805b081a4cfb93985d724953f4397c91b/netty-tcnative-boringssl-static-2.0.4.Final.jar
Archive:  ./.gradle/caches/modules-2/files-2.1/io.netty/netty-tcnative-boringssl-static/2.0.4.Final/b02dee1805b081a4cfb93985d724953f4397c91b/netty-tcnative-boringssl-static-2.0.4.Final.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2017-06-23 08:10   META-INF/
     1119  2017-06-23 08:10   META-INF/MANIFEST.MF
        0  2017-06-23 08:10   io/
        0  2017-06-23 08:10   io/netty/
        0  2017-06-23 08:10   io/netty/internal/
        0  2017-06-23 08:10   io/netty/internal/tcnative/
      357  2017-06-23 08:10   io/netty/internal/tcnative/Buffer.class
      743  2017-06-23 08:10   io/netty/internal/tcnative/CertificateRequestedCallback$KeyMaterial.class
      699  2017-06-23 08:10   io/netty/internal/tcnative/CertificateRequestedCallback.class
     9049  2017-06-23 08:10   io/netty/internal/tcnative/CertificateVerifier.class
     2966  2017-06-23 08:10   io/netty/internal/tcnative/Library.class
     3834  2017-06-23 08:10   io/netty/internal/tcnative/NativeStaticallyReferencedJniMethods.class
     1262  2017-06-23 08:10   io/netty/internal/tcnative/SessionTicketKey.class
      185  2017-06-23 08:10   io/netty/internal/tcnative/SniHostNameMatcher.class
     5660  2017-06-23 08:10   io/netty/internal/tcnative/SSL.class
     2985  2017-06-23 08:10   io/netty/internal/tcnative/SSLContext.class
      464  2017-06-23 08:08   META-INF/io.netty.versions.properties
        0  2017-06-23 08:10   META-INF/maven/
        0  2017-06-23 08:10   META-INF/maven/io.netty/
        0  2017-06-23 08:10   META-INF/maven/io.netty/netty-tcnative-boringssl-static/
    22970  2017-06-23 08:07   META-INF/maven/io.netty/netty-tcnative-boringssl-static/pom.xml
      136  2017-06-23 08:10   META-INF/maven/io.netty/netty-tcnative-boringssl-static/pom.properties
      245  2017-06-23 08:10   META-INF/INDEX.LIST
---------                     -------
    52674                     23 files

```

**Comment by normanmaurer** [comment]
Seems like I missed something during release process :/ will do another release, 😐

**Comment by normanmaurer** [comment]
@carl-mastrangelo sorry... released 2.0.5.Final which fixes this
